setup-ovs.sh

```diff
@@ -66,7 +66,6 @@ if [[ $(nmcli conn | grep -c ovs) -eq 0 ]]; then
       nmcli c delete $(nmcli c show |grep ovs-cnv |awk '{print $1}') || true
   else
       nmcli conn mod brcnv-iface connection.autoconnect yes
-      reboot
```

Comment:
can you add a short description on the commit message itself? 
Suggestion: 
```
removing reboot

removing reboot because recent changes in OCP/CoreOS cause the OVS config to get lost in between reboots
```

Reply:
I'm afraid I may have mislead you.

what I meant is that the first line of the commit should be `removing reboot` (aka commit summary) as you already did.
Then there should be an empty line and only then the commit description.

```
removing reboot

Removing the `reboot` since we need the bridge/bond to be created on every boot, not just a one time thing.```